dotcom-rendering/package.json

```diff
@@ -5,26 +5,25 @@
 	"license": "Apache-2.0",
 	"private": true,
 	"scripts": {
-		"preinstall": "../scripts/preinstall.mjs",
-		"lint": "yarn lint:src && yarn lint:cypress",
-		"lint:fix": "yarn lint:src --fix && yarn lint:cypress --fix",
+		"lint": "pnpm '/^lint:(cypress|scripts|src)$/'",
+		"lint:fix": "pnpm lint --fix",
```

Comment:
I don’t think this pases the arguments as expected

```
. lint:src$ eslint . --ext .ts,.tsx --quiet --cache "--fix"
└─ Running...
. lint:scripts$ eslint scripts --quiet --cache "--fix"
└─ Running...
. lint:cypress$ eslint ./cypress --cache "--fix"
└─ Done in 472ms
```

Reply:
this shouldn't affect how it works - iiuc, all options are strings that are then coerced (or not) into types by the whatever code constitutes the program you're running (in this case eslint).

i think the issue predates pnpm, in that the eslint command `ext` flags have not updates include all files in the application (`.mjs` etc – by default eslint only looks for `.js`).

i'm also not 100% sure why there's a rule for everything (`.`) and 2 for directories that will also be included in that rule (`cypress` and `scripts`)?

Reply:
Yeah all fair points, probably one to address separately!